src/Calculator/Calculator.vcxproj.filters

```diff
@@ -465,6 +465,7 @@
     <Page Include="Views\TitleBar.xaml">
       <Filter>Views</Filter>
     </Page>
+    <Page Include="$(MSBuildThisFileDirectory)DensityStyles\Compact.xaml" />
```

Comment:
><Page Include="$(MSBuildThisFileDirectory)DensityStyles\Compact.xaml" /> [](start = 4, length = 72)

did you add this manually?  If not, I'm curious as to how it was done.  I think this is probably not quite what you want to add to the filters file, but rather a relative path to the page and a Filter sub element to cover having the folder structure be preserved in visual studio.

Reply:
I did not add this manually.  I'm not sure how this was added.